samples/cpp/common/utils/include/samples/common.hpp

```diff
@@ -679,6 +679,15 @@ inline std::string getFullDeviceName(InferenceEngine::Core& ie, std::string devi
     }
 }
 
+inline std::string getFullDeviceName(ov::runtime::Core& ie, std::string device) {
```

Comment:
Can we delete previous getFullDeviceName(InferenceEngine::Core& ie, std::string device)   .
I think this method  is not used in the code anymore.

Reply:
i think we should leave this method for other possible calls

Reply:
it is needed for benchmark_app